treeherder/intermittents_commenter/commenter.py

```diff
@@ -63,41 +63,43 @@ def generate_bug_changes(self, startday, endday, alt_startday, alt_endday):
         for bug_id, counts in iteritems(bug_stats):
             change_priority = None
             bug_info = None
-            whiteboard = None
+            change_whiteboard = None
             priority = 0
             rank = None
 
-            # recommend disabling when more than 150 failures tracked over 21 days
```

Comment:
It's occurred to me that this should be moved to the bottom, so it doesn't risk getting reassigned in the weekly or daily sections.